atlassian_jwt_auth/contrib/django/decorators.py

```diff
@@ -46,15 +46,15 @@ def validate_asap_wrapper(request, *args, **kwargs):
     return validate_asap_decorator
 
 
-def requires_asap(issuers=None):
+def requires_asap(issuers=None, subject_should_match_issuer=True):
```

Comment:
Should we allow this to be governed by a django setting?

Reply:
For my use case I need to set it as an arg on the decorator factory, but I can make where if it is set to None (the default) it uses a Django setting.